keyboards/ibm/model_m_4th_gen/overnumpad_1xb/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+
+#Using STM32F446RET6
+MCU = STM32F446
```

Comment:
```suggestion
# MCU name
MCU = STM32F446 # STM32F446RET6

# Bootloader selection
BOOTLOADER = stm32-dfu
```

Reply:
Alright, also tested with explicitly specifying the bootloader, and that works fine too.